### ISSUE TITLE
Authentication issue

### ISSUE THREAD

**Issue by js1972** [inquaero/ABAPFire#3]
Hi. I notice you are only supporting email/pw as an authentication mechanism... Does this mean we cannot use a firebase service account (which only seems to work with token-based auth)?

I'm not sure how to find the necessary information:

ls_config-apikey = '[your apikey]'.
ls_config-authdomain = '[your authdomain]'.

Maybe firebase has moved on since these methods were possible?

**Comment by alborghetti** [comment]
Hi. You can get apikey and authdomain from your project in Firebase (just click on "Add Firebase to your web app".
Service account is not supported, as the main idea was to create a REST client helper in ABAP.

If you need admin SDK, probably the best way is to install a java or node server and connect them to your ABAP server.
At this link you can find more info on REST authentication:
[https://firebase.google.com/docs/reference/rest/auth/?authuser=0](https://firebase.google.com/docs/reference/rest/auth/?authuser=0)

**Comment by js1972** [comment]
Yes I thought as much - as in you're using user-level security. We need to use a service account instead. I've got it working easily enough with Python. The ABAP OAuth client is hideous however so I think I will just call out to a python script to get a JWT then use that to authenticate the API calls in ABAPfire.  ;-)
